deployments/apiv2/services/users/src/main.py

```diff
@@ -369,7 +387,10 @@ async def _send_registration_email(username: str, email: EmailStr, verification_
 
 
 @app.get("/")
-async def get_all_users(request: Request, token=Depends(ProtectedAny(scope=["users:admin"]))):
+async def get_all_users(
+    request: Request,
+    token=Depends(ProtectedAny(scope=PULSE3D_CUSTOMER_SCOPES)),
```

Comment:
Got it, yeah I didn't add anything yet for a super admin account for curibio.. and was thinking customers would have service scopes in case they didn't pay for all services so thought we'd just add `ProtectedAny(scope=[PULSE3D_CUSTOMER_SCOPES, <SERVICE>_CUSTOMER_SCOPES, ....]`

Reply:
is a customer account logged into the dashboard page specific to pulse3d or will it be attached to all the services?